transforms/code/license_select/python/Makefile

```diff
@@ -0,0 +1,56 @@
+
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../../..
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=license_select
+# $(REPOROOT)/.make.versions file contains the versions
+DOCKER_IMAGE_VERSION=${LICENSE_SELECT_PYTHON_VERSION}
```

Comment:
DOCKER_IMAGE_VERSION can be removed.

Reply:
@shivdeep-singh-ibm can you please check this input & address it